WalletWasabi.Documentation/WasabikaCodex.md

```diff
@@ -45,8 +45,43 @@ __Tips__: if the PR is too big ask for breakdown.
 
 __Fact__: code review is one of the few scientifically proven code improvement tools.
 
-## Creating Pull Request
+## How to fork a repository
+- On Github navigate to the desired repository.
+- In the top-right corner of the page, click Fork. ![Image of Fork Icon](https://docs.github.com/assets/images/help/repository/fork_button.jpg)
+
+### Cloning your forked repository
+- On GitHub.com, navigate to your fork. For Example: https://github.com/adampetho/WalletWasabi
+- Above the list of files, click the Code button with the arrow pointing down. ![Image of Code Button](https://docs.github.com/assets/images/help/repository/code-button.png)
+- Use one of the 3 options to get your repository into your local machine. (See below)
+
+#### Git Bash approach
+- Get and Setup Git Bash on your local computer
+- Copy the link
+- ![Image of Copy icon](https://docs.github.com/assets/images/help/repository/https-url-clone-cli.png)
+- Open Git Bash.
+- Change the current working directory to the location where you want the cloned directory.
+- Type git clone, and then paste the URL you copied earlier. Example: "git clone https://github.com/zkSNACKs/WalletWasabi.git"
+- Press Enter. Your local clone will be created.
+- Further reading about this approach or if you get stuck, please read this https://docs.github.com/en/get-started/quickstart/fork-a-repo (or ask the devs)
+
+#### Zip approach
+- Self-explanatory: Download -> Unzip -> Ready to go
+
+#### GitHub Desktop approach
+- Install GitHub Desktop and Login
+- Choose the Open with GitHub Desktop option
+- Choose where to clone
+- Click OK
 
+## Creating Pull Request
+- Fork the repo (See above)
+- Clone the forked repo (See above)
+- Make a new branch from master by "git checkout -b newbranch" (Replace newbranch with your own comprehensive names.)
```

Comment:
Given `checkout` is so overloaded the recommendation should be use `git switch -c newbranch`

Reply:
```suggestion
- Make a new branch from master by "git switch -c newbranch" (Replace newbranch with your own comprehensive names.)
```